Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenController.cs

```
@@ -0,0 +1,164 @@
+        }
+
+        private async UniTask ShowTipsAsync(CancellationToken ct)
+        {
+            tips = await sceneTipsProvider.Get(ct);
+
+            List<SceneTips.Tip> list = ListPool<SceneTips.Tip>.Get();
+
+            if (tips.Random)
+                tips.Tips.Shuffle(list);
+            else
+                list.AddRange(tips.Tips);
+
+            foreach (SceneTips.Tip tip in list)
+                viewInstance.AddTip(tip);
+
+            currentTip = 0;
+            viewInstance.ShowTip(currentTip);
+
+            ListPool<SceneTips.Tip>.Release(list);
+        }
+
+        private async UniTask WaitUntilWorldIsLoadedAsync(CancellationToken ct)
+        {
+            async UniTaskVoid UpdateProgressBarAsync(CancellationToken ct)
+            {
+                do
+                {
+                    try
+                    {
+                        float progress = Mathf.Clamp01(await inputData.AsyncLoadProcessReport.ProgressCounter.WaitAsync(ct));
+                        await UniTask.SwitchToMainThread(ct);
+                        viewInstance.ProgressBar.normalizedValue = progress;
+                        progressLabel!.Value = (int)(progress * 100);
+                    }
+                    catch (OperationCanceledException) { }
+                }
+                while (viewInstance.ProgressBar.normalizedValue < 1);
+            }
+
+            var progressUpdatingCancellationToken = CancellationTokenSource.CreateLinkedTokenSource(ct);
+            UpdateProgressBarAsync(progressUpdatingCancellationToken.Token).Forget();
+
+            try
+            {
+                await inputData.AsyncLoadProcessReport.CompletionSource.Task;
+                progressUpdatingCancellationToken.Cancel();
+                ct.ThrowIfCancellationRequested();
+                viewInstance.ProgressBar.normalizedValue = 1f;
+                progressLabel!.Value = 100;
+            }
+            catch { }
```

I think you should not hide all the exceptions